package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "chokidar '(client|server|test)/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
I'm sorry, it definitely wasn't obvious what the killing was for.  The killing isn't for when a dev is running two at a time.

`chokidar-cli` launches the test-runner each time a file is saved.  When someone saves a file twice, then the tests get run twice in parallel. The expected behavior is to abort the first test-run and only keep the second one.  This problem becomes much worse as someone runs this on a larger number of tests therefore increasing the likelihood that multiple test-runners will be working in parallel (and hogging tons of CPU / emitting strange output). 

You can test this out locally by running the test-watcher, and quickly saving a watched file N times.  If you run `top` you will see 2N `node` processes hogging CPU & RAM